tap_github/streams.py

```diff
@@ -179,11 +282,25 @@ def get_url_params(
 
     schema = th.PropertiesList(
         th.Property("id", th.IntegerType),
+        th.Property("node_id", th.StringType),
         th.Property("repo", th.StringType),
         th.Property("org", th.StringType),
-        th.Property("issue_number", th.IntegerType),
+        th.Property("issue_url", th.IntegerType),
```

Comment:
I think the `issue_number` property was previously coming from parent context. What do you think of adding back via `post_process()` method? 

Something like `"/".split(record["issue_url"])[-1]` might do the trick.

Reply:
I like that! I'll add it in. It does seem handy to have the issue number available.